Essentials/src/main/java/com/earth2me/essentials/Jails.java

```diff
@@ -318,5 +353,13 @@ public void onJailPlayerJoin(final PlayerJoinEvent event) {
                 future.completeExceptionally(ex);
             }
         }
+
+        private boolean shouldIgnore(final Player base) {
+            if (base.getUniqueId().version() <= 2) {
```

Comment:
Does this mean bedrock users can't be jailed?

Reply:
i do not care about geyser